docker-compose.yml

```diff
@@ -50,3 +50,22 @@ services:
     volumes:
       - .:/velox:delegated
     command: /bin/bash -c "scl enable gcc-toolset-9 '/velox/scripts/docker-command.sh'"
+
+  python:
```

Comment:
this looks unrelated to this change?

Reply:
Ah, yes I used this for testing these changes and found it useful. We can move into a separate PR if you prefer?

Reply:
I think that's ok. Just making sure this isn't leftover and we still want this merged?

Reply:
I will uses this in future testing so it will likely also be useful for others. :+1: